### ISSUE TITLE
Make sure all digits having leading number "01" not "1"

### ISSUE THREAD

**Issue by TimeScience** [borevitzlab/exif2timestream#30]
Folder names are inconsistently named... 
overstory_nadir-1
overstory_oblique-01

second one is correct

I'll verify this isn't an issue with the text in the CSV file

**Comment by jbraiuka** [comment]
Yup, this isn't an issue with the csv. its an issue with the code. ill get started on a fix, but for a quick / dirty workaround, where you put CAM_NUM in the ts_structure, if its a single digit cam_num, just replace it with:
...0CAM_NUM...
This will manually put the 0 in.
